testsuite/tests/properties/memoized_property_error/test.py

```diff
@@ -28,5 +28,9 @@ def prop2():
         return PropertyError(Bool, 'Explicit error 2')
 
 
-build_and_run(lkt_file='expected_concrete_syntax.lkt', ada_main='main.adb')
+build_and_run(lkt_file='expected_concrete_syntax.lkt', ada_main='main.adb',
+
+              # FIXME: internal errors without diagnostics when set to True,
```

Comment:
Could you replace FIXME with TODO? (to be consistent with the rest of the codebase, and thus make it easier to spot such issues with a grep)

Reply:
Done!